microsoft-edge/progressive-web-apps-chromium/how-to/service-workers.md

```diff
@@ -1,107 +1,106 @@
 
-Make sure to place the Service Worker file in the highest-level directory that you want it to manage.  Such a directory is called the _scope_ of the Service Worker.  In the previous code, the file is stored in the root directory of your app, and the Service Worker manages all pages that are under the app's domain name.
+Make sure to place the service worker file in the highest-level directory that you want it to manage.  Such a directory is called the _scope_ of the service worker.  In the previous code, the file is stored in the root directory of your app, and the service worker manages all pages that are under the app's domain name.
 
-If the Service Worker file was stored in a `js` directory, the scope of the Service Worker would be limited to the `js` directory and any subdirectories.  As a best practice, place the Service Worker file in the root of your app, unless you need to reduce the scope of your Service Worker.
+If the service worker file was stored in a `js` directory, the scope of the service worker would be limited to the `js` directory and any subdirectories.  As a best practice, place the service worker file in the root of your app, unless you need to reduce the scope of your service worker.
 
 
 <!-- ====================================================================== -->
 ## Intercept requests
 
-The main event that you use in a Service Worker is the `fetch` event. The `fetch` event runs every time the browser that your app runs in attempts to access content within the scope of the Service Worker.
+The main event that you use in a service worker is the `fetch` event. The `fetch` event runs every time the browser that your app runs in attempts to access content within the scope of the service worker.
 
 The following code shows how to add a listener for the `fetch` event:
 
 ```javascript
 self.addEventListener("fetch", event => {
-  console.log('WORKER: Fetching', event.request);
+  console.log("Fetching", event.request);
 });
 ```
 
 Within the `fetch` handler, you can control whether a request goes to the network, pulls from the cache, and so on.  The approach you take will likely vary, based on the type of resource being requested, how frequently it is updated, and other business logic that's unique to your application.
 
 Here are a few examples of what you can do within the `fetch` handler:
 
-*   If available, return a response from the cache; otherwise, fallback to requesting the resource over the network.
-*   Fetch a resource from the network, cache a copy, and return the response.
-*   Allow users to specify a preference to save data.
-*   Supply a placeholder image for certain image requests.
-*   Generate a response directly in the Service Worker.
+* If available, return a response from the cache; otherwise, fallback to requesting the resource over the network.
```

Comment:
debate: "fall back" vs. "fallback".
A fallback is a real technical thing.
The verb is: "fall back to x", or "use a fallback".

my inclination is "otherwise, fall back to".  OTOH, if this literally involves using a fallback, as a technical thing, "fallback to" could sort of be ok.

been wondering, not just for this PR.  @captainbrosset

Reply:
decided: changing to "fall back to"

Reply:
our dictionary says fallback is a noun, not a verb.
https://ahdictionary.com/word/search.html?q=fallback

Reply:
Let's avoid this altogether then: "If available, return a response from the cache; otherwise, request the resource over the network."